integrationTests/CreateRegularGivingMandateTest.php

```diff
@@ -95,6 +95,8 @@ public function setUp(): void
     public function tearDown(): void
     {
         $this->getContainer()->set(MessageBusInterface::class, $this->originalMessageBus);
+
+        parent::tearDown();
```

Comment:
not sure exactly why PHPStan suddenly started complaining that this is missing, but it's right that it should be there.